modules/microshift-accessing-cluster-remotely.adoc

```diff
@@ -48,9 +48,7 @@ Use the following procedure to access the {product-title} cluster from a remote
 $ chmod go-r ~/.kube/config
 ----
 
-.Verify {product-title} is running
-
-. As `user@workstation`, verify that {product-title} is running by running the following command:
+. As `user@workstation`, verify that {product-title} is running by entering the following command:
```

Comment:
Nit. Typically we format this as a verification step according to the Modular Documentation guide. Take a look at the [procedure template](https://raw.githubusercontent.com/redhat-documentation/modular-docs/main/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc) for an example. This is not a blocker.

```suggestion
.Verification
* As `user@workstation`, verify that {product-title} is running by entering the following command:
```

Reply:
Correct, I had it this way before but it looked weird with a #1 step procedure and and #1 step verification. Can we have a one-step verification without the step number?

Reply:
We can use an astericks for single-step procedures.